File: guardian/GuardianiOS/GuardianiOS/Models/Bid.swift
import Foundation

struct Bid: Identifiable, Codable {
    let id: UUID
    let bidderType: UserType
    let bidderId: UUID
    let title: String
    let description: String
    let availableLicenses: [LicenseType]
    let workLocation: Location
    let workRadius: Double
    let pricePerHour: Double
    let validUntil: Date
    var active: Bool
}

File: guardian/GuardianAndroid/app/src/main/java/com/guardian/android/data/models/Bid.kt
package com.guardian.android.data.models

import java.util.UUID

data class Bid(
    val id: UUID,
    val bidderType: UserType,
    val bidderId: UUID,
    val title: String,
    val description: String,
    val availableLicenses: List<LicenseType>,
    val workLocation: Location,
    val workRadius: Double,
    val pricePerHour: Double,
    val validUntil: java.time.Instant,
    val active: Boolean
)
